From 5d0570cb82fd9b243b650b1c9fd5b8ce9e930b16 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Thu, 09 Jan 2025 12:20:36 +0200
Subject: [PATCH] [FSW-6867] fs1: fix File palad_bloader not exist

Fix compilation error of fs1 pldm
File 'palad_bloader.bin' does not exist

Change-Id: I04bf091fb24dc551c5eb48f80bec871db07edf05
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/board/habanalabs/fs1_arc/fs1-post-image-spi-arc_global.sh b/board/habanalabs/fs1_arc/fs1-post-image-spi-arc_global.sh
index 92fb72e..aac6670 100755
--- a/board/habanalabs/fs1_arc/fs1-post-image-spi-arc_global.sh
+++ b/board/habanalabs/fs1_arc/fs1-post-image-spi-arc_global.sh
@@ -6,7 +6,7 @@
 
 SHA256_SIZE=32
 SHA256_CHKSUM=sha256sum
-BOOTLOADER_BINFILE=palad_bloader.bin
+BOOTLOADER_BINFILE=${HL_TARGET}_bloader.bin
 HL_BASE_PATH=${BR2_EXTERNAL}/board/habanalabs
 BOARD_BASE=${HL_BASE_PATH}/${HL_DEV}
 DEVICE_BASE=${HL_BASE_PATH}/${HL_DEV}
